pkgs/servers/krill/default.nix

```diff
@@ -0,0 +1,44 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, openssl
+, pkg-config
+, stdenv
+, Security
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "krill";
+  version = "0.9.4";
+
+  src = fetchFromGitHub {
+    owner = "NLnetLabs";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256:1w5s4vmr126k5lnxsv3hfxx1yz2c1gp3hmwfz5jn4lar78r5kidy";
+  };
+
+  cargoSha256 = "sha256:0ksh68giq3yd8k7vynxdl13ccysl6i5ag2nz9nknbnrvpv201c88";
+
+  buildInputs = [ openssl ] ++ lib.optional stdenv.isDarwin Security;
+  nativeBuildInputs = [ pkg-config ];
+
+  buildNoDefaultFeatures = true;
```

Comment:
Can you add a comment why this is needed?

Reply:
I took this from routinator, another package (maintained by someone else) from the same developers.

Reply:
pkgs/servers/routinator/default.nix

Reply:
I don't think that applies to this package necessarily as well. He passes the sock feature one line further down, looking at the Cargo.toml of routinator we can see that he's avoiding to build the `UI` feature.

Reply:
Good find. Adjusted it and it doesn't seem to impact the build.